cishouseholds/pipeline/survey_responses_version_digital_ETL.py

```diff
@@ -10,4 +11,9 @@ def digital_specific_transformations(df):
     df = df.withColumn("visit_id", F.col("participant_completion_window_id"))
     df = df.withColumn("visit_datetime", F.col("digital_enrolment_invite_datetime"))
     df = transform_survey_responses_generic(df)
+    df = assign_whichever_col_not_null(
```

Comment:
Could this function not be replaced with `F.coalesce`? The spec says that only one column should have a value, so coalesce would take the non-null value, if there are any